packages/shared/lib/clients/orchestrator.ts

```diff
@@ -584,8 +586,11 @@ export class Orchestrator {
                     await cancelling(syncId);
 
                     await clearLastSyncDate(syncId);
-                    const del = await recordsService.deleteRecordsBySyncId({ syncId });
-                    await logCtx.info(`Records for the sync were deleted successfully`, del);
+                    const syncConfig = await getSyncConfigBySyncId(syncId);
+                    for (const model of syncConfig?.models || []) {
+                        const del = await recordsService.deleteRecordsBySyncId({ syncId, connectionId, model });
+                        await logCtx.info(`Records for model ${model} were deleted successfully`, del);
+                    }
```

Comment:
maybe this change could have directly been encapsulated in the deleteRecordsBySyncId so it's transparent to the user and no duplicate code

Reply:
I thought about it but I prefer to have all functions in `records` to follow the same pattern, taking a connection and a model. I even thought about create a explicit `Partition` type.

Reply:
understandable in the model, could just be a helper on the side, but not pushing anything it's still manageable like this 👌🏻